docs/installation/configuration/config_file.md

```diff
@@ -184,6 +184,8 @@ The full specification of the file [is defined here](https://github.com/Cybercen
         config: {}
 
     datastore:
+      archive:
+        enabled: true
```

Comment:
do we want the archive to be enabled by default?

Reply:
I'll change it to be disabled by default.